### PR TITLE
Updating alpaca to latest release in github

### PR DESCRIPTION
**Pull request by sdavtaker**
If this PR updates an existing port, please uncomment and fill out this checklist:

- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [x] The "supports" clause reflects platforms that may be fixed by this new version
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by sdavtaker** [comment]
@microsoft-github-policy-service agree

File: ports/alpaca/portfile.cmake [hunk 1]
```diff
@@ -2,8 +2,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO p-ranav/alpaca
-    REF v0.2.0
-    SHA512 0ac2c1c4f8e0534319bf852bac3852ee3674db7b1a9eda30462821ec4c9ddeeb6ceff09ef5f16eed9131af6c357a09f2cb909a12ea2f135ca7d496d90ff1865d
+    REF v0.2.1
```
Comment (jimwang118) [inline ports/alpaca/portfile.cmake:5]:
```suggestion
    REF "v${VERSION}"
```

**Review by jimwang118 — CHANGES_REQUESTED** [review]

**Comment by jimwang118** [comment]
Note: I will be converting your PR to draft status. The above suggested changes are only recommendations. If you are willing to adopt them, please click "ready for review" after making the modifications. If you do not wish to make any changes, please click "ready for review" directly. That way, I can be aware that you've responded since you can't modify the tags.

**Comment by jimwang118** [comment]
You need to re-run this command: `./vcpkg x-add-version {portname} --overwirte-version` and then submit the result.

**Comment by sdavtaker** [comment]
Rerun the version with overwrite this time.

**Review by jimwang118 — APPROVED** [review]

**Comment by jimwang118** [comment]
Usage test pass with following triplets:

```
x86-windows
x64-windows
x64-windows-static
```

**Pull request merged**